### DISCUSSION TITLE
Sonoff SwitchMan 2C LED's

### DISCUSSION THREAD

**Discussion by Mark-Muc** [arendst/Tasmota#18066] [Q&A]
Hello out there!!
I am using SONOFF M5 SwitchMan 2 Channel (https://www.aliexpress.com/item/1005003618822308.html)
final Tasmota 12.4.0 (shutter)
Template from blackadder:
{"NAME":"Sonoff SwitchMan 2C","GPIO":[0,0,0,0,32,576,0,0,0,0,0,33,0,0,416,225,0,0,0,224,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],"FLAG":0,"BASE":1}
Everything works fine biond the LED for the second switch, if it is on .. NO LED goes on like it is for button 1.
For button 1 works like expected. but if the botton 2 is pressed . the relais goes on (fine) but now LED. If tried by a second one, also the same !!
LED are working in generel and I can dimm them ...
Anybody the same problem?
Thx & Regards
Mark

**Comment by barbudor** [comment]
The template only show 1 led configured as LedLink
So I would expect this meaning that any other led -if existing- would be directly connected to the relay control and not manageable by Tasmota

EDIT: I now see there is a PWM GPIO. Is that connected to a LED ?
What an idea...
At least that means that 2 leds are under control of Tasmota on GPIO5 and GPIO18
Up to you to configure them as you wish

**Comment by RafalNiewinski** [comment]
The answer is here https://github.com/arendst/Tasmota/discussions/15757
